README.md

```diff
@@ -118,6 +118,7 @@ The least we can do is to thank them and list some of their accomplishments here
 
 #### 2023
 * [Eric Everman](https://github.com/eeverman) added `@RestoreSystemProperties` and `@RestoreEnvironmentVariables` annotations to the [System Properties](https://junit-pioneer.org/docs/system-properties/) and [Environment Variables](https://junit-pioneer.org/docs/environment-variables/) extensions (#574 / #700)
+* [Florian Westreicher](https://github.com/meredrica) contributed to the JSON argument source extension.
```

Comment:
I think I would prefer to have the issue / PR listed afterwards (like elsewhere), even if it was closed.